<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bin2csv_qt</class>
 <widget class="QWidget" name="bin2csv_qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>748</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>748</width>
    <height>401</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>bin2csv</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Input directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInputDir">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>30</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInputDirBrowse">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutputFileBrowse">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutputFile">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>100</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConvert">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditConversionProgress">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>219</y>
     <width>721</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample rate</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSampleRate">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>170</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
